2018-06-26 22:58:18,937 [        MainThread][         start][    INFO] Starting TheRaidMap
2018-06-26 22:58:18,939 [        MainThread][       _legacy][    INFO] Starting factory <client.VNCDoToolFactory instance at 0x11682ccf8>
2018-06-26 22:58:19,169 [            VNCPic][       _legacy][    INFO] Using protocol version 3.800
2018-06-26 22:58:21,709 [        MainThread][   detect_text][    INFO] Check for Loginbutton - start ...
2018-06-26 22:58:22,011 [        MainThread][   detect_text][    INFO] Check for Loginbutton - finished...
2018-06-26 22:58:22,011 [        MainThread][   detect_text][    INFO] Check for Closebutton - start ...
2018-06-26 22:58:22,041 [        MainThread][         start][   ERROR] Uncaught exception
Traceback (most recent call last):
  File "start.py", line 104, in <module>
    main()
  File "start.py", line 68, in main
    check_message()
  File "/Users/fameitelhuber/Desktop/TheRaidMapper/detect_text.py", line 57, in check_message
    log.info(timer2)
UnboundLocalError: local variable 'timer2' referenced before assignment
